src/flux/store.ts

```diff
@@ -210,6 +206,12 @@ class StoreBase extends EventEmitter {
       case Constants.DELETE_FLOW:
         this.deleteFlow(payload);
         break;
+      case Constants.START_FLOW:
```

Comment:
Is this something to be moved to redux reducers?

Reply:
Initially this was written for flux but there is also a Redux implementation for this as well. As far as Jinad integration goes, we can remove any legacy flux code